src/library/styles/components/_components.scss

```diff
@@ -11,8 +10,13 @@
 @import 'link';
 @import 'product-card';
 @import 'modal';
+@import 'nav-primary/nav-primary';
```

Comment:
any reason we have nav-primary/nav-primary but not for others?

Reply:
The priamry nav styles are grouped in a folder because there are multiple stylesheets for nav primary.